erpnext/controllers/taxes_and_totals.py

```diff
@@ -50,16 +50,21 @@ def validate_conversion_rate(self):
 	def calculate_item_values(self):
 		if not self.discount_amount_applied:
 			for item in self.doc.get("items"):
+				price_list_rate = 0.0
 				self.doc.round_floats_in(item)
 
 				if item.discount_percentage == 100:
 					item.rate = 0.0
 				elif not item.rate:
-					item.rate = flt(item.price_list_rate *
+					price_list_rate = item.price_list_rate if item.uom == item.stock_uom \
```

Comment:
Why setting in a temporary variable? It should be set in price_list_rate field itself. Sometimes, user prints price_list_rate, in that case, it should be displayed as per sales uom.